java/src/com/here/flexpolyline/test/decoded.txt

```diff
@@ -0,0 +1,3072 @@
+{(1, 14, 2); [(-165.900000000000006, 79.099999999999994, 932.043507695096537), ]}
+{(1, 14, 3); [(40.299999999999997, -16.100000000000001, -795.325547749692760), ]}
+{(1, 14, 4); [(7.900000000000000, 88.900000000000006, 610.604874031274676), ]}
+{(1, 14, 5); [(-167.000000000000000, 82.599999999999994, 535.405899632264777), ]}
+{(1, 14, 6); [(-6.500000000000000, 71.200000000000003, -148.786816375532993), ]}
+{(1, 14, 7); [(109.700000000000003, -19.899999999999999, 292.387963103321056), ]}
+{(1); [(37.200000000000003, 57.100000000000001), ]}
+{(2, 13, 1); [(-100.530000000000001, -20.090000000000000, -783.570854149396951), ]}
+{(2, 13, 2); [(153.360000000000014, -67.790000000000006, -71.666489238810897), ]}
+{(2, 13, 3); [(126.180000000000007, 17.199999999999999, 666.522159882170172), ]}
+{(2, 13, 4); [(120.670000000000002, 40.170000000000002, 774.608401787319053), ]}
+{(2, 13, 5); [(-102.959999999999994, -21.649999999999999, -363.331171055102288), ]}
+{(2, 13, 6); [(101.379999999999995, 73.590000000000003, 801.595032687253479), ]}
+{(2, 13, 7); [(76.989999999999995, -62.649999999999999, -968.711536031164428), ]}
+{(2); [(-109.579999999999998, 19.820000000000000), ]}
+{(3, 12, 1); [(161.614000000000004, 31.370000000000001, 177.188143335314010), ]}
+{(3, 12, 2); [(-55.197000000000003, -71.269000000000005, -669.071372916466999), ]}
+{(3, 12, 3); [(111.209000000000003, -19.116000000000000, 556.410372880436967), ]}
+{(3, 12, 4); [(-7.842000000000000, 59.677999999999997, -866.037011797580021), ]}
+{(3, 12, 5); [(32.100999999999999, 13.680999999999999, -134.727188820344992), ]}
+{(3, 12, 6); [(-64.513999999999996, 34.932000000000002, -100.359269124402999), ]}
+{(3, 12, 7); [(-77.009000000000000, 59.170000000000002, -937.595069785072042), ]}
+{(3); [(-163.229999999999990, -46.962000000000003), ]}
+{(4, 11, 1); [(24.753599999999999, -52.970700000000001, -221.700311200019996), ]}
+{(4, 11, 2); [(-87.558700000000002, -2.271300000000000, -637.761106536970033), ]}
+{(4, 11, 3); [(35.411900000000003, -75.316500000000005, 804.016272289999961), ]}
+{(4, 11, 4); [(93.840500000000006, -41.533799999999999, -929.889005107379944), ]}
+{(4, 11, 5); [(159.418700000000001, 76.896799999999999, 58.025323505579998), ]}
+{(4, 11, 6); [(-15.819900000000001, 49.875799999999998, -856.212513040320005), ]}
+{(4, 11, 7); [(-5.832800000000000, -87.347200000000001, -713.729371309819953), ]}
+{(4); [(-75.180499999999995, 37.029499999999999), ]}
+{(5, 10, 1); [(-7.878950000000000, 83.812370000000001, 107.149447130200002), ]}
+{(5, 10, 2); [(157.096399999999988, -83.593249999999998, -246.103388527300012), ]}
+{(5, 10, 3); [(6.894930000000000, -16.158490000000000, 201.380364007299988), ]}
+{(5, 10, 4); [(-154.768900000000002, -9.018929999999999, 101.319675829299996), ]}
+{(5, 10, 5); [(-1.871230000000000, -65.574200000000005, 416.407619455100019), ]}
+{(5, 10, 6); [(-124.411230000000003, -76.537480000000002, -810.010750486600045), ]}
```

Comment:
Would it be possible to just re-use the same files for all language implementations? (even if it is just sym-linking them)

Reply:
Hello Victor, Thank you! for your help.

Reply:
Hello Victor,  I have made the changes that you have proposed. Kindly resolve it.